### PR TITLE
Update 3d rising bubble driver

### PR DESCRIPTION
**Pull request by sriharshakandala**
- [ ] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [ ] Unit tests are included OR N/A.
- [ ] Code is exercised in an integration test OR N/A.
- [ ] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: examples/hybrid/box/bubble_3d_invariant_rhoe.jl [hunk 1]
```diff
@@ -261,7 +311,7 @@ function rhs_invariant!(dY, Y, ghost_buffer, t)
     )
     @. dw -= vgradc2f(cp) / Ic2f(cρ)
 
-    cE = @. (norm(cuvw)^2) / 2 + Φ(z)
+    cE = @. (norm(cuvw)^2) / 2 + Φ(z, grav)
```
[the remark this answers is not in the document]
Comment (sriharshakandala) [reply examples/hybrid/box/bubble_3d_invariant_rhoe.jl]: The `norm` function seems to be allocating about 16 bytes! There are no other allocations in the `rhs_invariant!` function.

File: examples/hybrid/box/bubble_3d_invariant_rhoe.jl [hunk 2]
```diff
@@ -1,47 +1,125 @@
 using Test
+using Adapt
 using ClimaComms
-using StaticArrays, IntervalSets, LinearAlgebra, UnPack
+usempi = get(ENV, "CLIMACORE_DISTRIBUTED", "") == "MPI"
```
Comment (charleskawczynski) [inline examples/hybrid/box/bubble_3d_invariant_rhoe.jl:4]:
```suggestion
usempi = get(ENV, "CLIMACOMMS_CONTEXT", "") == "MPI"
```

**Comment by simonbyrne** [comment]
@sriharshakandala can you document what the arguments are in a comment at the top?

File: examples/hybrid/box/bubble_3d_invariant_rhoe.jl (hunk 2 above)
Comment (simonbyrne) [reply examples/hybrid/box/bubble_3d_invariant_rhoe.jl]: Or just use `ClimaComms.context()`

**Comment by simonbyrne** [comment]
bors try

**Comment by sriharshakandala** [comment]
> @sriharshakandala can you document what the arguments are in a comment at the top?

Done.

**Review by simonbyrne — APPROVED** [review]

**Comment by simonbyrne** [comment]
bors try

**Comment by simonbyrne** [comment]
bors try

**Comment by simonbyrne** [comment]
bors try

**Review by simonbyrne — APPROVED** [review]

**Pull request merged**